pkg/exporter/process.go

```diff
@@ -242,6 +242,7 @@ func (ep *ExportingProcess) createAndSendMsg(set entities.Set) (int, error) {
 	// than copying the set buffer to message buffer again.
 	bytesSlice := append(msg.GetMsgBuffer().Bytes(), set.GetBuffer().Bytes()...)
 	// Send the message on the exporter connection.
+	klog.Info(bytesSlice)
```

Comment:
debugging statement? If this is a must, please use increased log-level. Otherwise, you could remove it.

Reply:
Oops forgot to remove it. Thanks for reminding.